core/src/main/java/org/apache/calcite/util/format/FormatElementEnum.java

```diff
@@ -281,9 +301,9 @@ static Work get() {
 
     /** Uses Locale.US instead of Locale.ROOT to fix formatting in Java 11 */
     final DateFormat eeeeFormat = new SimpleDateFormat(DAY.javaFmt, Locale.US);
-    final DateFormat eeeFormat = new SimpleDateFormat(DY.javaFmt, Locale.ROOT);
-    final DateFormat mmmFormat = new SimpleDateFormat(MON.javaFmt, Locale.ROOT);
-    final DateFormat mmmmFormat = new SimpleDateFormat(MONTH.javaFmt, Locale.ROOT);
+    final DateFormat eeeFormat = new SimpleDateFormat(DY.javaFmt, Locale.US);
```

Comment:
Why use Locale.US?

Reply:
SQL requires the output to be in English. Using Locale.ROOT gives the wrong result for the eeeeFormat at least (Mon instead of Monday).

Reply:
It looks good to me